Gems/Atom/Feature/Common/Code/Include/Atom/Feature/CoreLights/ShadowConstants.h

```diff
@@ -27,7 +27,8 @@ namespace AZ
         static constexpr ShadowmapSize MinShadowmapImageSize = ShadowmapSize::Size256;
         static constexpr ShadowmapSize MaxShadowmapImageSize = ShadowmapSize::Size2048;
 
-        // This matchs ShadowFilterMethod in CoreLights/ViewSrg.azsli
+        // This matches ShadowFilterMethod in DirectionalLightShadowCalculator.azsli
+        // and represents m_shadowFilterMethod in CoreLights/ViewSrg.azsli
         enum class ShadowFilterMethod : uint32_t
```

Comment:
Why does this enum have one value, but the shader allows for 4 (None, Pcf, Esm, and EsmPcf)?

Reply:
Maybe there is a misunderstanding here but this enum has four values which should match the one in the shader